benchmark/cuchembench/datasets/base.py

```diff
@@ -28,12 +28,15 @@ def __init__(self,
             pathlib.Path(__file__).parent.parent.absolute(), 'csv_data', data_filename)
         assert os.path.exists(self.prop_data_path)
 
+        # prop_data = pd.DataFrame(columns=properties_cols)
+        # prop_data.to_csv(self.prop_data_path, index=False)
+        # #TODO cols must be passed in
         fp_data_path = os.path.join(
             pathlib.Path(__file__).parent.parent.absolute(), 'csv_data', fp_filename)
-        if not os.path.exists(fp_data_path):
-            logger.info(f'Fingerprint path {fp_data_path} does not exist, generating temporary fingerprints file.')
-            temp_dir = '/tmp' #tempfile.TemporaryDirectory().name
-            fp_data_path = os.path.join(temp_dir, self.fp_filename)
+        # if not os.path.exists(fp_data_path):
```

Comment:
Is the commented behaviour deprecated?

Reply:
The whole caching mechanism for fingerprints has been redesigned so we can ignore this file.